deploy/003_deploy_optimism_spokepool.ts

```diff
@@ -9,9 +9,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   // with deprecated spoke pool.
   // Set hub pool as cross domain admin since it delegatecalls the Adapter logic.
   const initArgs = [1_000_000, hubPool.address, hubPool.address];
-  await deployNewProxy("Optimism_SpokePool", initArgs, {
-    constructorArgs: ["0x4200000000000000000000000000000000000006", 3600, 32400],
-  });
+  await deployNewProxy("Optimism_SpokePool", ["0x4200000000000000000000000000000000000006", 3600, 32400], initArgs);
```

Comment:
It'd be nice to explicitly call out the constructor args -wdyt?

```suggestion
  const constructorArgs = ["0x4200000000000000000000000000000000000006", 3600, 32400];
  await deployNewProxy("Optimism_SpokePool", constructorArgs, initArgs);
```

Reply:
This was an existing pattern in the code. Do you think it should be changed in a separate PR?

Reply:
It's directly related to this change so I think we should just fix it in place :+1:

Reply:
Done